### ISSUE TITLE
Auth driver is not defined

### ISSUE THREAD

**Issue by SecretKeeper** [nfangxu/hyperf-http-auth#6]
hello , i have installed the package follow the instructions and also added 
```
    "fx/eloquent-provider": "^2.0",
    "hyperf/session": "^2.0",
    "fx/session-guard": "^2.0"
```
with and the `http-auth.php`:

```
'guards' => [
        'web' => [
            'driver' => 'session',
            'provider' => 'user-eloquent',
        ],
    ],
```

it throws 
`[ERROR] Auth driver [session] for guard [web] is not defined.[144] in /usr/share/nginx/html/hyperf/vendor/fx/hyperf-http-auth/src/HttpAuthManage.php`

**Comment by nfangxu** [comment]
fixed: 8c65537d232e37534921b19cde636d0e9d17c83f

**Comment by nfangxu** [comment]
fixed: v2.0.2
